src/app/cli/README.md

```diff
@@ -8,9 +8,11 @@ Use the OT Commissioner CLI to configure and manage OT Commissioner.
 ot-commissioner $ ./script/bootstrap.sh
 ot-commissioner $ mkdir build && cd build
 build $ cmake -GNinja -DCMAKE_BUILD_TYPE=Release ..
-build $ ninja -j16
+build $ ninja
```

Comment:
```suggestion
build $ ninja -j$(getconf _NPROCESSORS_ONLN)
```

Reply:
FYI: Raspberry Pi 3 has a quad-core ARM Cortex-A53 processor.

Reply:
@simonlingoogle I think command `ninja` without any arguments is doing exactly the same as what you commented. The problem is that a Raspberry Pi device may have two or four hardware threads but don't have enough memory to run two or four g++ compiling parallelly. Thus, I think we should better enforce single thread compiling on Raspberry Pi devices.

Reply:
Besides, I replaced all the build and configuration steps with the link to [openthread.io/ot-commissioner](https://openthread.google.cn/guides/commissioner/build). We will fix the documentation at there.

Reply:
> @simonlingoogle I think command `ninja` without any arguments is doing exactly the same as what you commented. The problem is that a Raspberry Pi device may have two or four hardware threads but don't have enough memory to run two or four g++ compiling parallelly. Thus, I think we should better enforce single thread compiling on Raspberry Pi devices.

Agree.